tools/mo/openvino/tools/mo/utils/class_registration.py

```diff
@@ -302,6 +307,8 @@ def apply_transform(graph: Graph, replacer_cls, **kwargs):
             for_graph_and_each_sub_graph_recursively(graph, lambda _: graph.check_shapes_consistency())
 
     except Error as err:
+        t = tm.Telemetry()
+        t.send_event('mo', 'error_info', "stage:{},transformation:{}".format(graph.stage, replacer_cls.__name__))
```

Comment:
why do we need to send it? Don't we duplicate the same one in `convert_impl.py`?